xml/admin_ceph_cephfs.xml

```diff
@@ -56,7 +56,8 @@
        The basic mount syntax is:
      </para>
 <screen>
-&prompt.cephuser;sudo mount -t ceph monip[:port][,monip2[:port]...]:/[subdir] mnt
+&prompt.root;mount -t ceph <replaceable>MON1_IP</replaceable>[:<replaceable>PORT</replaceable>],<replaceable>MON2_IP</replaceable>[:<replaceable>PORT</replaceable>],...:/<replaceable>OPTIONAL_SUBDIR</replaceable> \
```

Comment:
```suggestion
&prompt.root;mount -t ceph <replaceable>MON1_IP</replaceable>[:<replaceable>PORT</replaceable>],<replaceable>MON2_IP</replaceable>[:<replaceable>PORT</replaceable>],...:<replaceable>CEPHFS_MOUNT_TARGET</replaceable> \
```
Lets make it clear that a directory is needed after the last `:`. In most cases this `/` but I think this is more straight forward then saying you need `/` and an optional subdir (but without the `/`).

Reply:
good point :+1: